config/default.json

```diff
@@ -108,5 +108,9 @@
       "encryptionKey": "not-a-real-32-bit-encryption-key",
       "AppInsightsInstrumentationKey":"APPINSIGHTS_INSTRUMENTATIONKEY"
     }
+  },
+
+  "launchDarkly": {
+    "sdk": "sdk-AAAAA"
```

Comment:
this needs to go in the secrets block and the key name in vault needs to match this 
https://github.com/hmcts/cmc-citizen-frontend/blob/2a2987ca5509385505f28a601b4a7e6b9257cfd7/config/default.json#L100

Reply:
done